imagersite/django_imager/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for django_imager project.
+
+Generated by 'django-admin startproject' using Django 1.9.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.9/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+MEDIA_URL = '/media/'
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '!ma&n2r&@7q^82z&9e!cqkwfnu_ro-4#)4+c*#gxrtk5l@kao9'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'imager_images',
+    'imager_profile',
```

Comment:
Adding your apps to the top of `INSTALLED_APPS` will often cause problems, particularly with migrations. It's better practice to keep the default things that Django places into `INSTALLED_APPS` at the top of the list, and add all of yoru apps and everything else below.